HFStrategy/Strategy/Position.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+class Position(object):
+
+  def close_position(self, orderParams):
+    # some state usage here
+    pass
+
+  def open_position(self, orderParams):
+    pass
+  
+  def update_position(self, orderParams):
+    pass
+
+  def openShortPositionMarket(self, params):

```

Comment:
These will need to be statics to create a new position, but I like the idea of making a dedicated position class :dancer: